parser.js

```diff
@@ -2,99 +2,10 @@ var CompileError = require("./compileerror");
 var fs = require("fs");
 var path = require("path");
 
-// Clean up after solc.
-var listeners = process.listeners("uncaughtException");
-var solc_listener = listeners[listeners.length - 1];
-
-if (solc_listener) {
-  process.removeListener("uncaughtException", solc_listener);
-}
```

Comment:
You sure this is safe to remove?

Reply:
It's handled in the CompilerProvider now [here](https://github.com/trufflesuite/truffle-compile/pull/57).  But good catch, I think there's one missing place over there where we need to check. Going to add that to this PR.